RecoverVm/AttachOsDiskAsDataDiskToRecoveryVm.ps1

```diff
@@ -0,0 +1,63 @@
+﻿function AttachOsDiskAsDataDiskToRecoveryVm   (
+    [string] $ServiceName , 
+    [string] $VName ,    
+    [string] $RecoveryAdmin = 'recoveryAdmin',
+    [string] $RecoveryPW = [guid]::NewGuid().ToString()
+    )
+{
+    $RecoveryVMName = 'RC' + $(get-date -f yyMMddHHmm)    
+
+    #get defect vm object and export cfg to disk (temp)
+    $vm = Get-AzureVM -ServiceName $ServiceName -Name $VMName
+
+    if ( ! $vm ) 
+    {
+        return
```

Comment:
An ARM VM does not have a service name but a resource group name. If customers mix it up, they'll get an error that the service does not exist